### PR TITLE
Dropdown Menu of Analysis Types

### PR DESCRIPTION
**Pull request by danieldortega77**
Change the analysis types to a dropdown menu instead of a horizontal selection with Bootstrap, similar to #31

### CODE REVIEW

File: capstone/src/main/webapp/text_analyzer.html [hunk 1]
```diff
@@ -59,25 +59,36 @@ <h1>Text Analysis</h1>
...
-            </div>
-            <div class="form-check form-check-inline">
-              <input class="form-check-input" type="radio" name="analysisRadios" id="byWord" value="WORD">
-              <label class="form-check-label" for="byWord">By word</label>
-            </div>
-            <div class="form-check form-check-inline">
-              <input class="form-check-input" type="radio" name="analysisRadios" id="noAnalysis" value="NONE">
-              <label class="form-check-label" for="noAnalysis">No analysis</label>
-            </div>
+
```
Comment (chrismohrii) [inline capstone/src/main/webapp/text_analyzer.html]: nit: Let's remove this extra line and the one on line 91

**Review by chrismohrii — COMMENTED** [review]
LGTM

File: capstone/src/main/webapp/text_analyzer.html (hunk 1 above)
Comment (danieldortega77) [reply capstone/src/main/webapp/text_analyzer.html]: Oops, I forgot about this. Fixed!

**Review by chrismohrii — APPROVED** [review]

**Review by olu144 — APPROVED** [review]

**Review by dslucas — APPROVED** [review]

**Review by dborkan — APPROVED** [review]
LGTM

**Pull request merged**